src/tests/ov_helpers/ov_snippets_models/src/subgraph_mha.cpp

```diff
@@ -13,6 +13,42 @@
 namespace ov {
 namespace test {
 namespace snippets {
+namespace {
+std::vector<int64_t> rescale_order(std::vector<int64_t> default_order, size_t rank) {
```

Comment:
I saw quite similar functions in `mha_tokenization.hpp`. I'm just wondering: can we reuse them here?